health/micro-ui/web/micro-ui-internals/packages/modules/health-dss/src/components/MapChart.js

```
@@ -0,0 +1,196 @@
+  const aggregationRequestDto = {
+    visualizationCode: id,
+    visualizationType: "metric",
+    queryType: "",
+    requestDate: requestDate,
+    filters: { campaignId: campaignId },
+    aggregationFactors: null,
+  };
+  const { isLoading, data: response } = Digit.Hooks.DSS.useGetChartV2(aggregationRequestDto);
+
+  const onMouseEnter = (geo, current = { value: "0" }, event) => {
+    return settooltipContent(`${t(`${geo.properties.name}`)}: ${current.value ? Number(current.value).toFixed() + " ULBs" : "NA"} `);
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)